ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt

```diff
@@ -36,7 +44,12 @@ import kotlin.concurrent.thread
 data class ServerInfo(
     val pipeline: Pipeline,
     val port: Int,
-    var status: Pipeline.State
+    var status: Pipeline.State,
+)
+
+data class OffloadingServiceStatus(
+    val pipeline: Pipeline,
+    val registrationListener: RegistrationListener,
 )
```

Comment:
`Pipeline` and `RegistrationListener` needs type converts to use rooms.
https://developer.android.com/training/data-storage/room/referencing-data
I thought these objects would only be meaningful when the service was running, so I saved it as a map like before.
Would it be better to save them too?

Reply:
If those objects are meaningful only after instantiated, a map seems OK.  
> Would it be better to save them too?
Do you mean saving them to the DB?

Reply:
What happens to the already launched pipeline if the service is killed from external?

Reply:
> Do you mean saving them to the DB?

Yes

> What happens to the already launched pipeline if the service is killed from external?

Even if the pipeline is not closed in onDestroy, the client cannot access tensor_query_server.
I am going to check the detailed process.